keylime/registrar_client.py

```diff
@@ -45,6 +47,8 @@ def init_client_tls(config,section):
 
     if not config.getboolean('general',"enable_tls"):
         logger.warning("TLS is currently disabled, AIKs may not be authentic.")
+        global enableTLS
+        enableTLS = False
```

Comment:
don't set this here please, just use a pre run script with sed:

https://github.com/keylime/keylime/blob/7c6e0ea539d3952991e77eaefb57fbf14b83ea4e/test/run_tests.sh#L136-L137

Reply:
Ok, I'll follow that.